modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/StatusRenderer.js

```diff
@@ -15,12 +15,33 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+function getLabelType(label) {
```

Comment:
While this helper function would work perfectly fine as it is, it is better scoping if we place it inside the component.
```js
function StatusRenderer(props) {
    const getLabelType = (label) => {...}

    return props.value ?...
```

If it is in root, it should be either exported, or used by multiple components.

Reply:
True. Changed and I'll remember in the future